README.md

```diff
@@ -221,29 +227,31 @@ Las siguientes opciones de NixOS determinan qué certificados se aceptan o bloqu
 }
 ```
 
-Si `./mi-certificado.crt` está en la lista oficial, se incluirá en autofirma-nix.
-En cambio, dado que `Izenpe.com` se bloquea, se ignorará aunque aparezca en la lista oficial.
+If `./mi-certificado.crt` is on the official list, it will be included in autofirma-nix.
+However, since `Izenpe.com` is blocked, it will be ignored even if it appears on the
+official list.
 
-## Solución de problemas
+## Troubleshooting
 
-### Los dispositivos de seguridad no parecen actualizarse o no aparecen
+### Security devices do not seem to update or do not appear
 
-Si se ha instalado AutoFirma y se ha habilitado la integración con Firefox, pero
-Firefox no detecta los dispositivos de seguridad, es posible que sea necesario
-eliminar el fichero `pkcs11.txt` de la carpeta de perfil de Firefox. Por ejemplo,
-si se activa el módulo de Home Manager y su perfil se llama `miperfil`, el fichero
-se encontrará en `~/.mozilla/firefox/miperfil/pkcs11.txt`.
+If you have installed AutoFirma and enabled Firefox integration, but Firefox does not
+detect the security devices, you may need to remove the `pkcs11.txt` file from the
+Firefox profile folder. For instance, if you enabled the Home Manager module and the
+profile is named `miperfil`, the file is located in `~/.mozilla/firefox/miperfil/pkcs11.txt`.
 
-Bastará con eliminarlo y reiniciar Firefox.
+Removing it and restarting Firefox should solve the issue:
 
 ```console
 $ rm ~/.mozilla/firefox/miperfil/pkcs11.txt
 $ firefox
 ```
 
-### No aparecen los certificados aunque se haya solicitado el PIN del DNI-e
+### Certificates do not appear even though the DNIe PIN was requested
```

Comment:
Certificates do not appear -> Missing certificates

Co-authored-by: @CesarGallego